packages/f/fullPage.js.json

```
@@ -1,16 +1,27 @@
 {
-  "name": "fullPage.js",
+  "name": "fullpage.js",
```

```suggestion
  "name": "fullPage.js",
```
Must match file name